app/views/admin/timelogs/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= form_for [:admin, :user, @timelog] do |f| %>
+  <% if @timelog.errors.any? %>
+    <div class="error_explanation">
+      <ul>
+      <% @timelog.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <% unless @trello_service.cards.empty? %>
+    <div class="form-inputs">
+      <p><%= f.hidden_field :user_id, value: params[:user_id] %></p>
```

Comment:
you don't need this. All timelogs in controllers should be build/find by `@user`.